te_markdown/terminology.md

```diff
@@ -8,7 +8,33 @@ Test-Editor-Web is a lightweight application for specifying, implementing, and e
 
 ## Overview
 
-<!--- TODO: insert (annotated?) overview image and briefly describe the main areas of the user interface -->
+![Overview](/images/te.overview.svg?sanitize=true){:class="img-responsive"}
+
+### The different areas of the Test-Editor-Web
```

Comment:
I would suggest to remove this heading – you don't need it here, it follows directly after the previous heading without any text content in between (only the image), and it leads to very deep nesting levels. If you remove it here, you can "pull up" all the following sub-sections by one level (i.e. remove on "#" from each of them).

```suggestion
```